playground/src/components/Header.tsx

```diff
@@ -7,7 +7,7 @@ import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline'
 /** External links for the top navigation menu */
 const navigation = [
     { name: 'Documentation', href: 'https://wiki.polygon.technology/docs/miden/user_docs/assembly/main' },
-    { name: 'Examples', href: 'https://github.com/0xPolygonMiden/examples#available-examples' },
+    { name: 'Developer Tools', href: 'https://0xpolygonmiden.github.io/miden-vm/intro/usage/main.html' },
```

Comment:
I modified this in https://github.com/0xPolygonMiden/miden-vm/pull/814 and the new link will be `https://0xpolygonmiden.github.io/miden-vm/tools.html`